src/server/pfs/db/persist/clock.go

```diff
@@ -13,6 +13,18 @@ func (b *BlockRef) Size() uint64 {
 	return b.Upper - b.Lower
 }
 
+// ReadableCommitID returns a human-friendly commit ID for
+// displaying purposes.
+func (c *Clock) ReadableCommitID() string {
+	return fmt.Sprintf("%s/%d", c.Branch, c.Clock)
+}
```

Comment:
Since this function is defined on the `Clock` struct which is defined in the `persist` package, it will be hard.  Also the readable form is actually stored internally (in the database), and it feels like `pretty` is only for printing purposes.